hack/nfpm.yml

```diff
@@ -0,0 +1,30 @@
+name: docker-buildx
+
+# The architecture is specified using Go nomenclature (GOARCH) and translated
+# to the platform specific equivalent. In order to manually set the architecture
+# to a platform specific value, use deb_arch, rpm_arch and apk_arch.
+arch: ${ARCH}
+platform: linux
+
+version: ${VERSION}
+epoch: 0
+
+section: default
+
+maintainer: Docker <support@docker.com>
+description: Docker Buildx plugin extends build capabilities with BuildKit.
+vendor: Docker, Inc
```

Comment:
Maybe the vendor name should be provided via `--build-arg`?

Reply:
Hum yeah indeed, thanks for your feedback

Reply:
needs https://github.com/goreleaser/nfpm/pull/406 to expand env for vendor field or we could sed but that's kinda dirty.